### ISSUE TITLE
Problem with `new_entity_secondary_time_index` in dfs

### ISSUE THREAD

**Issue by Seth-Rothschild** [Featuretools/featuretools#58]
For the dataset I'm using, I have
```
es.normalize_entity('coordinates', 'plows', 'truck_name', 
                    make_time_index=True,
                    make_secondary_time_index={'date_fixed': []})
```
so that I can automatically calculate the first and last time that a plow is on the road. If I give the secondary time index a new name, with `new_entity_secondary_time_index`
```
es.normalize_entity('coordinates', 'plows', 'truck_name', 
                    make_time_index=True,
                    make_secondary_time_index={'date_fixed': []},
                    new_entity_secondary_time_index='last_coordinates_time')
```
then I get a `KeyError` in dfs. This seems to come from the line in `_filter_and_sort`:
```
    618             if time_last is not None and not df.empty:
--> 619                 mask = df[secondary_time_index] >= time_last
    620                 second_time_index_columns = self.secondary_time_index[secondary_time_index]
    621                 df.loc[mask, second_time_index_columns] = np.nan
```

**Comment by Seth-Rothschild** [comment]
It looks like `es['plows'].secondary_time_index` stays as `{'date_fixed': 'date_fixed'}` even after `new_entity_secondary_time_index`. 

In fact, `new_entity_secondary_time_index` does
```
            if new_entity_secondary_time_index:
                secondary_df.rename(columns={secondary_time_index:new_entity_secondary_time_index},
                                    inplace=True)
```
which only changes the columns of the underlying dataframe. Setting `new_entity.secondary_time_index` explicitly would fix

**Comment by Seth-Rothschild** [comment]
Closed with #59
